snappy/overlord.go

```diff
@@ -460,10 +460,10 @@ func (o *Overlord) InstallWithSideInfo(snapFilePath string, sideInfo *snap.SideI
 	}
 
 	// deal with the data
-	err = CopyData(newInfo, oldInfo, flags, meter)
+	err = copyData(newInfo, oldInfo, flags, meter)
```

Comment:
we are close to be able to, but it needs a lot of test surgery in snappy so probably best as its own branch, also not clear it's valuable work if the end result anyway is removing snappy